lib/perl/Bio/EnsEMBL/Pipeline/Runnable/BRC4/DumpGenomeSQL.pm

```diff
@@ -50,7 +50,7 @@ sub fetch_input {
   }
   
   # Dump file
-  my $dump_file = $organism ? $organism . ".sql.gz" : $species . "sql.gz";
+  my $dump_file = $organism ? $organism . ".sql.gz" : $species . ".sql.gz";
```

Comment:
Unless you are planning on these to have different extensions, you could also do:
```suggestion
  my $dump_file = ($organism ? $organism : $species) . ".sql.gz";
```